symplyphysics/laws/thermodynamics/compressibility_factor_via_intermolecular_force_potential.py

```diff
@@ -0,0 +1,89 @@
+from sympy import Eq, Integral, pi, exp, S, Piecewise
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+# Description
+## The virial equation describes the deviation of a real gas from ideal gas behaviour. The virial coefficients
+## in the virial expansion account for interactions between successively larger groups of molecules. Since
+## interactions between large numbers of molecules are rare, the virial equation is usually truncated at the third
+## term onwards. Under the assumption that only pair interactions are present, the compressibility factor can be
+## linked to the intermolecular force potential:
+
+# Law: Z = 1 + 2 * pi * N / V * Integral((1 - exp(-phi(r) / (k * T))) * r**2, (r, 0, infinity))
+## Z - [compressibility factor](../../definitions/compressibility_factor_is_deviation_from_ideal_gas.py)
+## N - number of particles
+## V - volume
+## r - radius
+## phi(r) - intermolecular force potential as a function of radius
```

Comment:
Shouldn't it depend on the distance between molecules?

Reply:
Yes, you are right